package.json

```diff
@@ -77,15 +77,15 @@
     "karma-sourcemap-loader": "^0.3.8",
     "karma-webpack": "^5.0.0",
     "less": "^4.1.1",
-    "less-loader": "^7.3.0",
+    "less-loader": "^10.0.0",
     "less2sass": "^1.0.3",
     "npm-run-all": "^4.0.1",
     "postcss": "^8.2.6",
-    "source-map-loader": "^2.0.1",
+    "source-map-loader": "^3.0.0",
     "style-loader": "^2.0.0",
-    "ts-loader": "^8.3.0",
-    "typedoc": "^0.20.28",
-    "typescript": "^4.2.2",
+    "ts-loader": "^9.2.3",
+    "typedoc": "^0.21.0",
+    "typescript": "^4.3.4",
```

Comment:
is the new override config causing versioning conflicts in the type definitions?

Reply:
Override keyword does not appear in type definitions. I assume that is what you are referring to.